static/docs/user-guide/external-data/gs.md

```diff
@@ -0,0 +1,77 @@
+# External Data on Google Cloud Storage
+
+Google Cloud Storage is a RESTful online file storage web service for storing
+and accessing data on Google Cloud Platform infrastructure.
+
+## External Dependencies
+
+Let's take as an example a stage that simply downloads a file from a GS
+location:
+
+```dvc
+$ dvc run \
+      -d gs://mybucket/file.csv \
+      -o file.csv \
+      'gsutil cp gs://mybucket/file.csv file.csv'
+```
+
+It is even easier to download a file if we use the command `dvc import-url`:
+
+```dvc
+$ dvc import-url gs://mybucket/file.csv
+```
+
+Or, using a remote:
+
+```dvc
+$ dvc remote add gs-data gs://mybucket
+$ dvc import-url remote://gs-data/file.csv
+```
+
+## External Data and Outputs
+
+For cached external outputs (specified using `-o`) we need to setup an external
+cache location that will be used by DVC to store versions of the external file:
+
+```dvc
+# Add GS remote to be used as cache location for GS files
+$ dvc remote add gscache gs://mybucket/cache
+
+# Tell dvc to use the 'gscache' remote as GS cache location
+$ dvc config cache.gs gscache
+```
+
+> Non-cached external outputs (specified using `-O`) do not require an external
+> cache to be setup.
+
+<details>
+
+### Warning: Don't use the same location for the external cache and for the DVC storage
+
+When you setup an external cache for your external outputs, avoid using the same
```

Comment:
I don't like that we repeat over and over again. It feels like we can put into index page, general overview page or something?